### PR TITLE
README.md: Add `Usage` section

### PR DESCRIPTION
**Pull request by arafsheikh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ This package offers a plugin for the Eclipse to use the code analysis software [
 mvn clean verify
 ```
 
+## Usage
+To learn how to use the plug-in please refer to the [Using the plug in]
+(https://github.com/coala-analyzer/coala-eclipse/wiki/Using-the-plug-in) wiki page.
```
Comment (Adrianzatreanu) [inline README.md]: are you sure? wikis are often outdated and never read first. i think you should rather explain in here briefly and give more explanations there

**Comment by Adrianzatreanu** [comment]
the yellow color of the pending review label is misleading, can you change it to the green we are using :p ? this one is for wips :D

File: README.md (hunk 1 above)
Comment (arafsheikh) [inline README.md]: @harshdattani thoughts?

**Comment by sils** [comment]
@arafsheikh how's this going?

**Comment by aptrishu** [comment]
Still working on this ? @arafsheikh 
It has been a long time we haven't seen any update from you.

**Comment by sils** [comment]
araf?

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala.io/ - http://viperdev.io/ - http://gitmate.com/

**Comment by arafsheikh** [comment]
Closing in favor of #23

**Pull request closed without merging**